cmd/influxd/inspect/verify_seriesfile_test.go

```diff
@@ -9,33 +9,41 @@ import (
 	"testing"
 	"time"
 
-	"github.com/influxdata/influxdb/v2/cmd/influx_inspect/verify/seriesfile"
 	"github.com/influxdata/influxdb/v2/models"
 	"github.com/influxdata/influxdb/v2/tsdb"
+	"github.com/stretchr/testify/require"
 	"go.uber.org/zap"
 )
 
+func TestVerifies_BasicCobra(t *testing.T) {
+	test := NewTest(t)
+	defer os.RemoveAll(test.Path)
+
+	verify := NewVerifySeriesfileCommand()
+	verify.SetArgs([]string{"--dir", test.Path})
+	require.NoError(t, verify.Execute())
+}
+
 func TestVerifies_Valid(t *testing.T) {
 	test := NewTest(t)
-	defer test.Close()
+	defer os.RemoveAll(test.Path)
 
-	verify := seriesfile.NewVerify()
+	verify := NewVerify()
 	if testing.Verbose() {
 		verify.Logger, _ = zap.NewDevelopment()
```

Comment:
I think we could set up the tests to always set `verify.Logger = zaptest.NewLogger(t)` and get this behavior for free. It'll buffer any logs and only print them if the test fails or if running in verbose mode.